src/ident/MessageId.php

```diff
@@ -10,7 +10,17 @@
 class MessageId
 {
     /**
-     * Create a new message id.
+     * @var Referenece Refers to a session at a specific revision.
```

Comment:
Actually, leave this - I want to see if it gets picked up by phpstan once #7 is done.